MySolutionUnitTest.rb

```diff
@@ -0,0 +1,56 @@
+require 'minitest/autorun'
+
+require_relative 'MySolution'
+
+class MySolutionUnitTest < Minitest::Test
+
+    def setup
+        @unit = MySolution.new
+    end
+
+#problem1
+    def test_sumOfEven_ordinaryArray
+        array = [1,2,3,4,5,6]
+        assert_equal 12 , @unit.sumOfEven(array)
+    end
+
+    def test_sumOfEven_onlyEvenNumbers
+        array = [2,6,18]
+        assert_equal 26, @unit.sumOfEven(array)
+    end
+
+    def test_sumOfEven_onlyOddNumbers
+        array = [3,5,7]
+        assert_equal 0,@unit.sumOfEven(array) 
+    end
+
+#problem2
+    def test_getLargest2Numbers
+        array = [5, 9, 1, 19, 14, 3, 10, 25, 9]
+        result = [19,25]
+        assert_equal result,@unit.getLargest2Numbers(array)
+    end
+
+    def test_getLargest2Numbers_BigMaxFirst
+        array = [5, 9, 1, 19, 14, 3, 10, 25, 9]
+        result = [19,25]
+        assert_equal result,@unit.getLargest2Numbers(array)
+    end
+
+    def test_getLargest2Numbers_negativeValues
+        array = [-3, -2]
```

Comment:
Does this test work if you put set the array to: [-2, -3]?

Reply:
Yes it passes